lib/futbol_data.rb

```diff
@@ -101,9 +81,48 @@ def create_coach_by_team_id(season)
     end
   end
 
+  # Used in League
```

Comment:
This is a good spot for this :)

Reply:
Yay! I was hoping this was a good use of CSV behavior versus statistics behavior